website/content/partials/cli/monitor/flags/log-level.mdx

```diff
@@ -0,0 +1,9 @@
+<a id="monitor-flag-log-level" />
+
+**`-log-level (string: "info")`**
```

Comment:
```suggestion
**`-log-level (enum : info)`**
```

While it's implemented as a string, *technically* the type is an `enum` because there are explicit tokens expected for the flag value and you don't need to use quotes with the value